test/jit/test_module_containers.py

```diff
@@ -237,18 +237,7 @@ def __init__(self):
             def forward(self, v):
                 return self.mods[-11].forward(v)
 
-        with self.assertRaisesRegex(Exception, "Index -11 out of range"):
-            torch.jit.script(M2())
-
-
-        class M2(M):
```

Comment:
Wait can you delete this if it's accessed in line 241? Or is it accessing another M2